custom-connectors/EleadCustomerSales/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1909 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Elead Sales Customers",
+    "version": "1.0",
+    "description": "This API allows for the management of prospect/customer information inside the Elead CRM.This API includes the following capabilities and features for managing prospect/customer information, Add prospect/customer information such as name, phone numbers, email addresses and physical address and return the uniquely-assigned Elead ID on post, Update, search and retrieve prospect/customer contact information tied to an existing Elead ID.",
```

Comment:
There is a missing space after the first sentence's period and the second sentence of the description, between `CRM.` and `This`